prebid-mobile/pbm-api/android/pbm-adunit-android.md

```diff
@@ -47,12 +47,62 @@ PB Ad Slot is an identifier tied to the placement the ad will be delivered in. T
 
 Trigger a call to Prebid Server to retrieve demand for this Prebid Mobile ad unit.
 
+#### Mopub & GAM
+
+By default, Prebid SDK uses inflection to determine the publisher ad server, one of Mopub or Google Ad Manager (GAM), to convert the Prebid's targeting keys (PBS bid keys, host and cache key) to trigger targeted line items. To render ads in ad servers other than Mopub or GAM, use the next section's 3rd party ad server support feature.
```

Comment:
>>one of Mopub or Google Ad Manager (GAM)<<

Do you mean to say either MoPub or Google Ad Manager (GAM)

Reply:
Good catch. It is or. One cannot operate both at the same time. I will update it.